pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -66,11 +66,6 @@ let
   # source tree.
   extraAttrs = buildFun base;
 
-  githubPatch = { commit, sha256, revert ? false }: fetchpatch {
```

Comment:
I guess maintainers might want to keep stuff like this if they regularly use it.